code/modules/clothing/shoes/sneakers.dm

```diff
@@ -5,7 +5,7 @@
 	greyscale_config = /datum/greyscale_config/sneakers
 	greyscale_config_worn = /datum/greyscale_config/sneakers_worn
 	flags_1 = IS_PLAYER_COLORABLE_1
-	greyscale_config_worn_digi = /datum/greyscale_config/sneakers_worn/digi //SKYRAT EDIT ADDITION
+	greyscale_config_worn_digi = /datum/greyscale_config/sneakers_worn/digi  //SKYRAT EDIT - DigiGreyscale
```

Comment:
Any reason why you made it have two spaces before the two slashes? Also, same thing as for the previous comment, won't repeat it for all of them but it should be easy enough to bulk-fix with VSC.

Reply:
Probably messed up while copy-pasting the Skyrat Edit - DigiGreyscale line x-x its inconsistent in random places, fixed now